pages/index.rst

```diff
@@ -10,6 +10,8 @@
 
 What is HPy?
 ============
+Pronounced /h/ - pie
```

Comment:
The `-----` makes a subsection in ReStructured Text. I would recommend just making this sentence a new paragraph after the "HPy provides ..." one below.